compiler/rustc_lint/messages.ftl

```diff
@@ -406,6 +406,15 @@ lint_non_fmt_panic_unused =
     }
     .add_fmt_suggestion = or add a "{"{"}{"}"}" format string to use the message literally
 
+lint_non_local_definitions = non-local `impl` definition, they should be avoided as they go against expectation
```

Comment:
oh, wait. this should say `macro` instead of `` `impl` `` for macros.

Reply:
good find, fixed!